templates/validationGeneric.hbs

```diff
@@ -1,3 +1,4 @@
+{{#unless (isMultipartSchema .)}}
```

Comment:
This is quite interesting... we should really _only_ be generating these methods _if_ the content type is JSON eh?